linkerd/app/core/src/config.rs

```diff
@@ -37,19 +33,6 @@ pub struct ProxyConfig {
     pub detect_protocol_timeout: Duration,
 }
 
-/// A `HashSet` specialized for ports.
-///
-/// Because ports are `u16` values, this type avoids the overhead of actually
-/// hashing ports.
-pub type PortSet = HashSet<u16, BuildHasherDefault<PortHasher>>;
-
-/// A hasher for ports.
-///
-/// Because ports are single `u16` values, we don't have to hash them; we can just use
-/// the integer values as hashes directly.
-#[derive(Default)]
-pub struct PortHasher(u16);
```

Comment:
Not really a part of `config` in general -- this has been moved the `port_policies` module as an impl detail.